backend/src/main/java/com/darass/darass/auth/oauth/controller/AuthenticationPrincipalArgumentResolver.java

```diff
@@ -31,6 +28,11 @@ public boolean supportsParameter(MethodParameter parameter) {
     public User resolveArgument(MethodParameter parameter, ModelAndViewContainer mavContainer,
         NativeWebRequest webRequest, WebDataBinderFactory binderFactory) {
         String accessToken = AuthorizationExtractor.extract(Objects.requireNonNull(webRequest.getNativeRequest(HttpServletRequest.class)));
-        return authService.findUserByToken(accessToken);
+
+        if (accessToken == null) {
```

Comment:
```suggestion
        if (Objects.isNull(accessToken)) {
```
null 하드 코딩보다는 이미 있는 메소드를 사용해 보는 것은 어떨까요?

Reply:
반영하겠습니다. 😃